specs/windows/etw_process_events.table

```diff
@@ -0,0 +1,29 @@
+table_name("etw_process_events")
```

Comment:
(This isn't blocking)

Does it make sense to expose any of this in the existing `process_events`? I feel like we've never really had a good answer for how we handle that, and the migration between event subsytems. (process_events vs bpf_process_events vs es_process_events)

I guess maybe it makes more sense to let them all diverge, and people can write platform specific SQL?

Reply:
My takeaway from office hours was:
* We should keep separate tables
* Future work to explore a merged `process_events`
* We should probably name them `process_events_etw` so like tables sort together

Reply:
@marcosd4h thanks for the context! This is what I get for not coming to OH :)

Reply:
https://hackmd.io/F9uUgvUkQuuyfrPoZbstYg?view#ETW-events-table--Process-events

I think we were more nuanced.  IIRC the extended schema has some issues under the hood, and it's often hard to see how to combine things.

So we were mulling on the idea of having divergent tables, and _also_ a single table with a simplified set of things. But we didn't think we should block here on that.

We did most think we should start naming them `process_events_XXXX` so they sorted together

Reply:
@directionless @zwass I ended up naming the table `process_etw_events`. The main rationale for picking this name instead of `process_events_XXXX` is that evented tables must have the `_events `suffix at the end of the table name